fe/src/components/RatingsHistory/RatingsHistoryChart.re

```diff
@@ -4,13 +4,29 @@ open EloTypes;
 
 let component = statelessComponent("RatingsHistoryChart");
 
+let sumRatingWithDiff = (rating: ratingHistory): int => {
+  rating.userRating + rating.ratingDiff;
+};
+
+let calculateActualRating = (ratingsHistory: list(ratingHistory)): int => {
+  ratingsHistory |> List.hd |> sumRatingWithDiff;
+};
+
+let addActualRating = (ratingsHistory, historyList) => {
```

Comment:
@ktrzos 
Tutaj to jest według mnie mało czytelne, że przekazujesz najpierw tą pełną pierwoną listę a potem tą przetwarzaną.
Wydaje mi się że czytelniej by było jakby w funkcji make, na początku było
`let actualRating = ratingsHistory |> calculateActualRating;`
a potem w pipeline:
`addActualRating(actualRating)`
Ale się nie upieram, merguje, a jak będziesz chciał to się jeszcze zmieni

Reply:
@ktrzos 
Albo jeszcze inaczej i chyba najlepiej by było jakby przenieść addActualRating przed mapą wtedy wogle mógłbyś bez arugmentowo to zrobić, by było
`let data =
      ratingsHistory
      |> addActualRating
      |> List.map(item => {"name": item.date, "rating": item.userRating})
...`

i Wtedy funkcja addActualRating by miała jeden argument ratingsHistory i by sobie mogła sama obliczyć actual Rating

Reply:
Hm, ok. Zaraz ogarnę :).

Reply:
@ktrzos 
Niewiem czy chcesz to jeszcze zmieniać, jak coś to daj znać to mogę mergować.

Reply:
@kamilkuz 
Jeśli jednak chciałbym bezargumentowo, to musiałbym zbudować identyczną strukturę jaka jest w `ratingsHistory`, co nie? Niepotrzebne dane musiałbym dodawać. A w momencie, jakby ten typ się rozszerzył, to kolejne kolumny musiałbym uzupełniać.

Reply:
@ktrzos 
Tak, to prawda, musiałbyś wtedy dodawać pełną strukturę ratingHistory. Więc w sumie tak jak jest teraz jest ok.